frontend/src/pages/WorkOrderPage.js

```diff
@@ -0,0 +1,240 @@
+import React from 'react';
+import { View, Picker, Text, TextInput, Switch, Button, KeyboardAvoidingView } from 'react-native';
+import RNPickerSelect from 'react-native-picker-select';
+import { createWorkOrder } from '../apis/workOrders/CreateWorkOrder'
+import { connect } from 'react-redux';
+import { workOrderPage } from '../stylesheets/WorkOrderPageStyleSheet'
+
+class WorkOrderPage extends React.Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            property_id: '5dc398c35c942a071043e187',
+            sector: 'ROOF',
+            type: 'IMP', 
+            title: 'untitled', 
+            cause: 'no cause', 
+            service_needed: false, 
+            priority: 'MEDIUM', 
+            description: 'no description', 
+            due_date: '2020-11-07T03:54:52.130+00:00',
+            price_estimate: 0
+        }
+    }
+    static navigationOptions = {
+        title: 'Create Work Order',
+    };
+    
+    handleWorkOrder = async() => {
+        try {
+            await createWorkOrder(
+                this.state.property_id,
+                this.state.sector,
+                this.state.type,
+                this.state.title,
+                this.state.cause,
+                this.state.service_needed,
+                this.state.priority,
+                this.state.description,
+                this.state.due_date,
+                this.state.price_estimate).then(async() => {
+                        this.props.navigation.goBack(null)
+                });
+        } catch (err) {
+            console.log(JSON.stringify(err.response))
+            console.log('' + this.state.property_id + '\n' + 
+                this.state.sector + '\n' +
+                this.state.type + '\n' + 
+                this.state.title + '\n' + 
+                this.state.cause + '\n' + 
+                this.state.service_needed + '\n' + 
+                this.state.priority + '\n' + 
+                this.state.description + '\n' + 
+                this.state.due_date + '\n' + 
+                this.state.price_estimate + '\n' +
+                err.response.data);
+        }
+    }
+
+    toggleCorrective = () => {
+        if(this.state.type == 'CM'){
+            this.setState({type: 'IMP'})
+        }
+        else{
+            this.setState({type: 'CM'})
+        }
+
+    }
```

Comment:
why if CM then set to IMP?

Reply:
if neither Preventive or Corrective is selected, it makes it IMP
if it is already Corrective (CM) and you click on it, it untoggles it (ie sets it to IMP).